django_select2/widgets.py

```diff
@@ -5,14 +5,18 @@
 import logging
 from itertools import chain
 import re
-import util
+import six
+from . import util
 
 from django import forms
 from django.core.validators import EMPTY_VALUES
-from django.utils.encoding import force_unicode
 from django.utils.safestring import mark_safe
 from django.core.urlresolvers import reverse
 from django.utils.datastructures import MultiValueDict, MergeDict
+try:
+    from django.utils.encoding import force_unicode
+except ImportError:
+    from django.utils.encoding import force_str as force_unicode
```

Comment:
I was symploy following what django was using for alias: 

```python
#django.utils.encoding.py

if six.PY3:
    smart_str = smart_text
    force_str = force_text
else:
    smart_str = smart_bytes
    force_str = force_bytes
    # backwards compatibility for Python 2
    smart_unicode = smart_text
    force_unicode = force_text
```

I am fine either way, I'll push a commit to change it.